amkotlinutil/src/main/java/dk/adaptmobile/amkotlinutil/extensions/ImageViewExtensions.kt

```diff
@@ -46,20 +47,15 @@ fun ImageView.loadImageResource(imageResource: Drawable?, skipMemoryCache: Boole
 }
 
 private fun ImageView.loadImage(imageResource: Any, skipMemoryCache: Boolean, transformation: Transformation?, imageLoadedCallback: (() -> Unit)?) {
-    var requestOptions = RequestOptions()
+    val glide = Glide.with(context)
+            .load(imageResource)
             .skipMemoryCache(skipMemoryCache)
 
-    requestOptions = when (transformation) {
-        is Transformation.CenterCrop -> requestOptions.centerCrop()
-        is Transformation.Circle -> requestOptions.circleCrop()
-        else -> requestOptions // Do nothing
+    when (transformation) {
```

Comment:
Check release notes for reference: https://github.com/bumptech/glide/releases/tag/v4.9.0